### PR TITLE
Created and updated docs for 1.28

### PR DESCRIPTION
**Pull request by bhavi-koduru**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by markapruett — APPROVED** [review]

**Review by markapruett — APPROVED** [review]

File: projects/kubernetes-csi/external-snapshotter/README.md
Comment (bhavi-koduru) [inline projects/kubernetes-csi/external-snapshotter/README.md:1]: Will update the snapshotter docs in a seperate PR to avoid conflicts

**Pull request merged**